### PR TITLE
policygen: support forseti iam domain policy

### PR DESCRIPTION
**Pull request by xingao267**

### CODE REVIEW

File: templates/policygen/forseti/org/iam_allow_domains.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+metadata:
+  name: iam_allow_domains
+spec:
+  severity: high
+  match:
+    target: ["organization/{{.org_id}}"]
+  parameters:
+    domains:
+    - gserviceaccount.com # always allow GCP service accounts
+{{- range get . "allowed_policy_member_domains"}}
```
Comment (umairidris) [inline templates/policygen/forseti/org/iam_allow_domains.yaml:26]: indent?

File: examples/policygen/config.yaml [hunk 2]
```diff
@@ -17,7 +17,11 @@ template_dir: ../../templates/policygen
 org_id: "12345678"
 
 # Optional fields
-forseti_policies: {}
+forseti_policies:
+  # TODO(xingao): consider merging with same attributes in org policies
+  allowed_policy_member_domains:
+  - google.com
```
Comment (umairidris) [inline examples/policygen/config.yaml:23]: example.com

Comment (xingao267) [reply examples/policygen/config.yaml]: Done

File: templates/policygen/forseti/org/iam_allow_domains.yaml (hunk 1 above)
Comment (xingao267) [reply templates/policygen/forseti/org/iam_allow_domains.yaml]: Done

**Pull request merged**